### DISCUSSION TITLE
Help on installation

### DISCUSSION THREAD

**Discussion by Infiniteial0** [winstxnhdw/lc-hax#28] [Q&A] [answered]
Hello and Sorry to bother,

I viewed the installation steps. I started off by recursively clone the repository. After that I opened the requirements.bat located in the lc-hax folder. But I have an issue within the cmd.
![image_2023-12-21_155902028](https://github.com/winstxnhdw/lc-hax/assets/154475768/53a2cb12-a44e-4c20-94c1-cf1c0a83a20c)
After that, I can click any key and it closes out.

**Comment by winstxnhdw** [comment, marked as answer]
There's no issue. Just wait till it finishes building. It may take a while. Afterwards, just launch the game and execute `launch.bat`.

**Reply by Infiniteial0** [reply]
Oh ok. Sorry to bother. I was just worried about the "Cannot convert null literal"
Tysm for the help.

**Reply by winstxnhdw** [reply]
Those are just lint warnings. I'll deal with them next time.
